Unable to build: aggregate ‘QPainterPath path’ has incomplete type and cannot be defined

<!-- IF YOU DO NOT USE THE TEMPLATE BELOW, YOUR ISSUE WILL BE CLOSED! THIS APPLIES FOR BUG REPORTS AND FEATURE REQUESTS, NO EXCEPTIONS!

The template below shows what you need to include in a good bug report, and you must use it.
I'm happy to help, but you have to ensure I fully understand what you want and have the information I need. -->

**Environment**
 - OS/Distribution: Arch Linux
 - Desktop Environment: <!-- if applicable, also whether you use X or Wayland, e.g. Gnome on Wayland, Xmonad on X -->
 - Hardware: <!-- for desktop: your GPU and which driver you use, for Android: your device, e.g. NVIDIA GeForce GTX 1080 with nouveau, Pixel 3 -->
 - Exact Version or Commit Hash and Installation Method: v1.2.1

**Describe the bug**
It seems with latest master, I'm unable to build the project, when attempting, I get following error:
```sh
/home/invidian/repos/chiaki/gui/src/servericonwidget.cpp: In lambda function:
/home/invidian/repos/chiaki/gui/src/servericonwidget.cpp:65:16: error: aggregate ‘QPainterPath path’ has incomplete type and cannot be defined
   65 |   QPainterPath path;
         |                ^~~~
         make[2]: *** [gui/CMakeFiles/chiaki.dir/build.make:250: gui/CMakeFiles/chiaki.dir/src/servericonwidget.cpp.o] Error 1
         make[1]: *** [CMakeFiles/Makefile2:392: gui/CMakeFiles/chiaki.dir/all] Error 2
         make: *** [Makefile:183: all] Error 2
```

It seems following patch fixes the build:
```diff
diff --git gui/src/servericonwidget.cpp gui/src/servericonwidget.cpp
index e9d4e08..f7d60d7 100644
--- gui/src/servericonwidget.cpp
+++ gui/src/servericonwidget.cpp
@@ -18,6 +18,7 @@
 #include <servericonwidget.h>

 #include <QPainter>
+#include <QPainterPath>

 ServerIconWidget::ServerIconWidget(QWidget *parent) : QWidget(parent)
 {
```

--- a/gui/src/servericonwidget.cpp
+++ b/gui/src/servericonwidget.cpp
@@ -18,6 +18,7 @@
 #include <servericonwidget.h>
 
 #include <QPainter>
+#include <QPainterPath>
 
 ServerIconWidget::ServerIconWidget(QWidget *parent) : QWidget(parent)
 {

